rio/srcrobot/commands/QueueCommand.py

```diff
@@ -0,0 +1,27 @@
+from commands2 import Command, ParallelCommandGroup, InstantCommand, SequentialCommandGroup
+from constants import Constants
+from robotState import RobotState
+from commands.TurnInPlace import TurnInPlace
+from subsystems.Arm import Arm
+from subsystems.indexer import Indexer
+from subsystems.Shooter import Shooter
+from subsystems.Swerve import Swerve
+from wpilib import DriverStation
+
+class QueueCommand(SequentialCommandGroup):
+    def __init__(self, shot: Constants.NextShot):
```

Comment:
Do we need this class at all? How about something like this in RobotContainer...
```
self.queSubFront.onTrue(InstantCommand(self.robotState.setNextShot(Constants.NextShot.SPEAKER_CENTER)))
```